spec/features/proposals_spec.rb

```diff
@@ -43,7 +43,7 @@
 
       click_button 'New'
       click_link "accept_event_#{@event.id}"
-      expect(flash).to eq('Event accepted!')
+      expect(page.has_content?('Event accepted!')).to be true
       expect(page.has_content?('Unconfirmed')).to be true
```

Comment:
@lagartoflojo 
> Sure, it can be another commit. But if someone is working on a file that would benefit from some unrelated cleaning, I think people should go for it (boyscout rule: Always leave the campground cleaner than you found it).

I am not saying that this shouldn't be changed, just that should be in another commit, in favour of a clean git history :smile: